serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/basic-integration-tests-with-restassured.adoc

```diff
@@ -1 +1,243 @@
-= Basic Integration Test with RestAssured
\ No newline at end of file
+= Basic Integration Test with REST Assured
+// Metadata:
+:description: Basic Integration Test with REST Assured
+:keywords: kogito, workflow, quarkus, serverless, quarkus-cli, test, restassured
+
+As a developer, you can use https://rest-assured.io/[REST Assured] to test your Serverless Workflow application,
+which includes the following procedures:
+
+* <<boostrapping-the-project,Bootstrapping a project>>
+* <<creating-workflow,Creating a workflow>>
+* <<testing-application,Testing your Serverless Workflow application>>
+
+Also, you can directly access an example application by cloning `kogito-quarkus-examples/serverless-workflow-testing-with-rest-assured`
+from the https://github.com/kiegroup/kogito-examples.git[Kogito Examples] GitHub repository.
+
+.Prerequisites
+* https://adoptium.net/[JDK {java_min_version}] is installed with `JAVA_HOME` configured appropriately.
+* https://maven.apache.org/install.html[Apache Maven] is installed.
+* https://quarkus.io/guides/cli-tooling[Quarkus CLI] is installed.
+* Visual Studio Code with https://marketplace.visualstudio.com/items?itemName=redhat.java[Red Hat Java Extension]
+and https://marketplace.visualstudio.com/items?itemName=redhat.vscode-extension-serverless-workflow-editor[Red Hat Serverless Workflow Editor] is installed to edit your workflows.
+
+For more information about the tooling and the required dependencies, see xref:getting-started/getting-familiar-with-our-tooling.adoc[tooling guide].
+
+This guide describes how to test the Serverless Workflow application that exposes the `hello` endpoint.
+The endpoint expects a `name` as a parameter and returns a `greeting`.
+
+Example: For a request with the given body:
+
+[source,json]
+----
+{
+  "workflowdata": {
+    "name": "John Doe"
+  }
+}
+----
+
+The endpoint should return a response with a body similar to:
+
+[source,json]
+----
+{
+  "id": "00ce5c1d-4936-4e18-998c-f5c5d5d784ad",
+  "workflowdata": {
+    "greeting": "Hello, John Doe"
+  }
+}
+----
+
+[[boostrapping-the-project]]
+== Bootstrapping a project
+
+To create your Serverless Workflow service you need to bootstrap a project. The example below is just to illustrate the testing example. Please see the xref:getting-started/create-your-first-workflow-service.adoc[Create Your First Serverless Workflow Service guide] for more information.
+
+.Procedure
```

Comment:
Shouldn't we just refer to the getting started guide for people to get a working example first?  So this guide would start with the assumption that you have a working SW project, either because you executed the getting started guide, you opened up an example (for example the one you reference here at the top), or they have their own working example.  And the guide just starts by saying what you need to do to add a test.  The example test you show will match the example on kogito-examples, but users should be able to do something similar for the other use cases.  Wdyt?

Reply:
Thanks for the review, @krisv.
I pushed the requested change.